geode-web/src/test/java/org/apache/geode/management/internal/cli/commands/CommandOverHttpDUnitTest.java

```diff
@@ -1,38 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
- * agreements. See the NOTICE file distributed with this work for additional information regarding
- * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance with the License. You may obtain a
- * copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- */
-package org.apache.geode.management.internal.cli.commands;
-
-import org.junit.ClassRule;
-import org.junit.contrib.java.lang.system.ProvideSystemProperty;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-import org.junit.runners.Suite;
-
-import org.apache.geode.test.junit.categories.DistributedTest;
-import org.apache.geode.test.junit.categories.GfshTest;
-import org.apache.geode.test.junit.categories.SecurityTest;
-import org.apache.geode.test.junit.runners.SuiteRunner;
-
-/**
- * Only commands that extends CliCommandTestBase needs to be in this test suite
- */
-@Category({DistributedTest.class, SecurityTest.class, GfshTest.class})
-@RunWith(SuiteRunner.class)
-@Suite.SuiteClasses({GemfireDataCommandsDUnitTest.class, ShellCommandsDUnitTest.class})
```

Comment:
this test is being run. We need to refactor those tests to not to use CliCommandBase and then get rid this test suite.

Reply:
I'd be in favor over removing the other empty suites, but I think this one needs to get converted into something that will actually run in CI. Maybe just create a separate ticket for that one?

Reply:
this test suite is run under Flaky category since both tests are flaky. And the test result is shown under each individual test inside the suite.

Reply:
@jinmeiliao - I see. I was looking in the wrong place. It is getting run - http://files.apachegeode-ci.info/builds/1.8.0-build.1112/test-results/flakyTest/1531246575/classes/org.apache.geode.management.internal.cli.commands.GemfireDataCommandsDUnitTest.html

Reply:
I've been bamboozled!  I thought they weren't getting run since they appear as
```
Class | Tests | Failures | Ignored | Duration | Success rate
-- | -- | -- | -- | -- | --
CommandOverHttpDUnitTest | 0 | 0 | 0 | - | -
```
in the class-level report (e.g., [this](http://files.apachegeode-ci.info/builds/1.8.0-build.1112/test-results/flakyTest/1531246575/packages/org.apache.geode.management.internal.cli.commands.html)).

So it's the empty ones that are actually just running tests redundantly that should be removed.

Since that's not a one-off, I'll go open a separate ticket for that.